specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json

```diff
@@ -363,6 +491,27 @@
         "storageResourceUri"
       ]
     },
+    "ValidationParameters": {
```

Comment:
These model names are odd. This one is generic and paired with "pre-backup", while `PreRestoreOperationalParameters` is very specific and tied to the name obviously as "pre-restore". You should go for some consistency here for a better REST API and SDK design.